packages/vscode-plugin/src/tests/suite/integration.test.ts

```diff
@@ -18,6 +20,7 @@ describe('Integration >', () => {
 	let markdownUri: Uri;
 
 	beforeAll(async () => {
+		await closeAll();
```

Comment:
Just curious, what's being closed here? Since this is the first test to run, shouldn't there be no tabs open at all?

Reply:
It may be a macOS thing where window states are saved.[^1] When I tested a bunch of times, the tabs from the previous test stayed, interfering with the new test.

[^1]: https://www.mothersruin.com/software/Archaeology/reverse/appstate.html

Reply:
> It may be a macOS thing where window states are saved.[1](#user-content-fn-1-4e23ce261a044025ff34942684b8e0ec) When I tested a bunch of times, the tabs from the previous test stayed, interfering with the new test.

This was my guess too by the way, and I don't think it's Mac-only thought I can only test Mac at the moment.